docker-compose.yml

```diff
@@ -16,6 +16,15 @@ services:
     networks:
       - frontend
       - backend
+  mailhog:
```

Comment:
Can we move the update from the root docker-compose into the `compose` folder?

Reply:
Ok, no problem.